app/views/payouts/list.njk

```diff
@@ -83,12 +82,39 @@ Payments to your bank account - GOV.UK Pay
       </tbody>
     </table>
   {% else %}
-    <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
-    {% if payoutsReleaseDate %}
-    <p class="govuk-body">No payments to your bank account found on or after {{ payoutsReleaseDate.format('D MMMM YYYY') }}.</p>
-    <p class="govuk-body">For historic payments to your bank account please <a class="govuk-link" href="https://www.payments.service.gov.uk/support/">contact support.</a></p>
+    {% if filterLiveAccounts %}
+      <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+      {% if payoutsReleaseDate %}
+        <p class="govuk-body">No payments to your bank account found on or after {{ payoutsReleaseDate.format('D MMMM YYYY') }}.</p>
+        <p class="govuk-body">For historic payments to your bank account please <a class="govuk-link" href="https://www.payments.service.gov.uk/support/">contact support.</a></p>
+      {% else %}
+        <p class="govuk-body">No payments to your bank account found.</p>
+      {% endif %}
     {% else %}
-    <p class="govuk-body">No payments to your bank account found.</p>
+      <h2 class="govuk-heading-m">Sample Report</h2>
+
```

Comment:
Some text missing here about sample report which reads `This is a sample report .......`

Reply:
Fixed.